src/modules/endpoint/DatabaseConnector.js

```diff
@@ -159,6 +159,10 @@ class DatabaseConnector {
     return `${CACHE_DIR_PATH}/cities.json`
   }
 
+  async deleteAllFiles () {
```

Comment:
If this is a private method then it should be prefixed with a `_`

Reply:
It's not private.